docs/components/multiselect.md

```diff
@@ -119,22 +163,22 @@ You cannot add an item if the `label` matches the `label` of a pre-existing item
     v-model="mySelections"
     :items="items"
     enable-item-creation
-    @item:added="(item) => trackNewItems(item, true)"
-    @item:removed="(item) => trackNewItems(item, false)"
+    @item-added="(item) => trackNewItems(item, true)"
+    @item-removed="(item) => trackNewItems(item, false)"
```

Comment:
why is this event changing? It previously followed the same convention as built-ins such as `@update:modelValue`